docs/back-end-api/erc20CollateralToken/graphql.md

```diff
@@ -707,13 +674,14 @@ Upon successful completion of a request, the server will issue a status code of
 
 ### `Repay`
 
-Repay a loan to a pool.
+Repay a loan to a pool if is not close and it is not already repaid.
```

Comment:
Repay a loan to a pool if `it` is not closed and it `has` not already `been` repaid.